transactions/src/main/java/org/jboss/as/txn/subsystem/TransactionSubsystem31Parser.java

```diff
@@ -0,0 +1,209 @@
+/*
```

Comment:
I think I have addressed this in my latest push. I extend 30Parser and just override one of the methods that parses the new attribute - is that what you meant @wolfc